Exiled.API/Features/Items/Scp244.cs

```diff
@@ -54,6 +54,22 @@ public bool Primed
             set => Base._primed = value;
         }
 
+        /// <summary>
+        /// Gets or sets the Scp244's remaining health.
```

Comment:
```suggestion
        /// Gets or sets the health of Scp244.
```

Reply:
This is wrong, it should be unchanged but the "the" before "Scp244's" should be removed